api-client/src/tracking/index.js

```diff
@@ -69,8 +69,7 @@ export type OpenLinkSignalType = {|
     | OpenExternalLinkSignalNameType
     | OpenOsLinkSignalNameType
     | OpenMediaSignalNameType,
-  url: string,
-  fromUrl: string
+  url: string
```

Comment:
Did you update the signal spreadsheet for documentation?

Reply:
Yes :+1: